packages/noya-utils/src/__tests__/sortBy.test.ts

```diff
@@ -3,3 +3,83 @@ import { sortBy } from '../index';
 test('empty array', () => {
   expect(sortBy([], '')).toEqual([]);
 });
+
+test('first key', () => {
+  expect(
+    sortBy(
+      [
+        { name: 'Liam', last: 'Smith' },
+        { name: 'Noah', last: 'Benjamin' },
+        { name: 'Sophia', last: 'Brown' },
+        { name: 'Emma', last: 'Jones' },
+        { name: 'Ava', last: 'Davis' },
+      ],
+      'name',
+    ),
+  ).toEqual([
+    { name: 'Ava', last: 'Davis' },
+    { name: 'Emma', last: 'Jones' },
+    { name: 'Liam', last: 'Smith' },
+    { name: 'Noah', last: 'Benjamin' },
+    { name: 'Sophia', last: 'Brown' },
+  ]);
+});
+
+test('second key', () => {
+  expect(
+    sortBy(
+      [
+        { name: 'Liam', last: 'Smith' },
+        { name: 'Noah', last: 'Benjamin' },
+        { name: 'Sophia', last: 'Brown' },
+        { name: 'Emma', last: 'Jones' },
+        { name: 'Ava', last: 'Davis' },
+      ],
+      'last',
+    ),
+  ).toEqual([
+    { name: 'Noah', last: 'Benjamin' },
+    { name: 'Sophia', last: 'Brown' },
+    { name: 'Ava', last: 'Davis' },
+    { name: 'Emma', last: 'Jones' },
+    { name: 'Liam', last: 'Smith' },
+  ]);
+});
+
+test('same value', () => {
+  expect(
+    sortBy(
+      [
+        { name: 'Liam', last: 'Smith' },
+        { name: 'Liam', last: 'Brown' },
+        { name: 'Ava', last: 'Davis' },
+      ],
+      'name',
+    ),
+  ).toEqual([
+    { name: 'Ava', last: 'Davis' },
+    { name: 'Liam', last: 'Smith' },
+    { name: 'Liam', last: 'Brown' },
+  ]);
+});
+
+test('empty space', () => {
+  expect(
+    sortBy(
+      [
+        { group: 'cloud', ammount: 9 },
```

Comment:
I think we can remove `ammount` here. And shorten the array to just 2 items, one with an empty string and one without